### PR TITLE
OSDOCS3082: Networking Operators

### PR DESCRIPTION
**Pull request by xenolinux**
[Preview](https://deploy-preview-45515--osdocs.netlify.app/openshift-enterprise/latest/networking/networking-operators-overview.html)

QE contact: @zhaozhanqi 

OCP version: 4.6, 4.7, 4.8, and 4.9 only
There is a different [PR](https://github.com/openshift/openshift-docs/pull/45517) for 4.10+ versions

[Jira](https://issues.redhat.com/browse/OSDOCS-3082)

### CODE REVIEW

**Comment by zhaozhanqi** [comment]
CNO part lgtm

**Comment by lihongan** [comment]
DNS operator and Ingress operator parts look good. Thank you!

File: networking/networking-operators-overview.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+[id="networking-operators-overview"]
+= Networking operators overview
```
Comment (pneedle-rh) [inline networking/networking-operators-overview.adoc:3]: You could capitalise "operators" in this title. Please see [this section of the guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-name-capitalization) regarding the capitalisation of "Operator".

**Review by pneedle-rh — COMMENTED** [review]
@xenolinux I have now completed my review of this PR. Please let me know if you have any questions about my suggestions.

Additionally, you could consider updating the commit comment for this PR to match the commit comment in https://github.com/openshift/openshift-docs/pull/45517, so that it is consistent across the branches.

Thanks!

File: _topic_maps/_topic_map.yml [hunk 2]
```diff
@@ -980,6 +980,8 @@ Topics:
   File: understanding-networking
 - Name: Accessing hosts
   File: accessing-hosts
+- Name: Networking operators overview
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply _topic_maps/_topic_map.yml]: Ack'ed

File: networking/networking-operators-overview.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+include::_attributes/common-attributes.adoc[]
+:context: networking-operators-overview
+
+toc::[]
+
+{product-title} supports multiple types of networking Operators. You can manage the {product-title} cluster networking using these networking Operators.
+
+[id="networking-operators-overview-cluster-network-operator"]
+== Cluster Network Operator
+The Cluster Network Operator (CNO) deploys and manages the cluster network components on an OpenShift Container Platform cluster, including the Container Network Interface (CNI) default network provider plug-in selected for the cluster during installation. For more information, see xref:../networking/cluster-network-operator.adoc#cluster-network-operator[Cluster Network Operator in {product-title}].
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply networking/networking-operators-overview.adoc]: Ack'ed

[the remark this answers is not in the document]
Comment (xenolinux) [reply networking/networking-operators-overview.adoc]: Ack'ed

File: networking/networking-operators-overview.adoc [hunk 4]
```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+[id="networking-operators-overview"]
+= Networking operators overview
+include::_attributes/common-attributes.adoc[]
+:context: networking-operators-overview
+
+toc::[]
+
+{product-title} supports multiple types of networking Operators. You can manage the {product-title} cluster networking using these networking Operators.
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply networking/networking-operators-overview.adoc]: Ack'ed

File: networking/networking-operators-overview.adoc [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+[id="networking-operators-overview-cluster-network-operator"]
+== Cluster Network Operator
+The Cluster Network Operator (CNO) deploys and manages the cluster network components on an OpenShift Container Platform cluster, including the Container Network Interface (CNI) default network provider plug-in selected for the cluster during installation. For more information, see xref:../networking/cluster-network-operator.adoc#cluster-network-operator[Cluster Network Operator in {product-title}].
+
+[id="networking-operators-overview-dns-operator"]
+== DNS Operator
+The DNS Operator deploys and manages CoreDNS to provide a name resolution service to pods, enabling DNS-based Kubernetes Service discovery in OpenShift. xref:../networking/dns-operator.adoc#dns-operator[DNS Operator in {product-title}].
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply networking/networking-operators-overview.adoc]: Ack'ed

File: networking/networking-operators-overview.adoc [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+[id="networking-operators-overview-ingress-operator"]
+== Ingress Operator
+When you create your OpenShift Container Platform cluster, pods and services running on the cluster are each allocated their own IP addresses. The IP addresses are accessible to other pods and services running nearby but are not accessible to outside clients. The Ingress Operator implements the IngressController API and is the component responsible for enabling external access to OpenShift Container Platform cluster services. For more information, see xref:../networking/ingress-operator.adoc#configuring-ingress[Ingress Operator in {product-title}].
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply networking/networking-operators-overview.adoc]: Ack'ed

[the remark this answers is not in the document]
Comment (xenolinux) [reply networking/networking-operators-overview.adoc]: Ack'ed

[the remark this answers is not in the document]
Comment (xenolinux) [reply networking/networking-operators-overview.adoc]: Ack'ed

[the remark this answers is not in the document]
Comment (xenolinux) [reply networking/networking-operators-overview.adoc]: Ack'ed

File: networking/networking-operators-overview.adoc [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+{product-title} supports multiple types of networking Operators. You can manage the cluster networking using these networking Operators.
+
+[id="networking-operators-overview-cluster-network-operator"]
+== Cluster Network Operator
+The Cluster Network Operator (CNO) deploys and manages the cluster network components in an {product-title} cluster. This includes deployment of the Container Network Interface (CNI) default network provider plug-in selected for the cluster during installation. For more information, see xref:../networking/cluster-network-operator.adoc#cluster-network-operator[Cluster Network Operator in {product-title}].
+
+[id="networking-operators-overview-dns-operator"]
+== DNS Operator
+The DNS Operator deploys and manages CoreDNS to provide a name resolution service to pods. This enables DNS-based Kubernetes Service discovery in {product-title}. For more information,  xref:../networking/dns-operator.adoc#dns-operator[DNS Operator in {product-title}].
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply networking/networking-operators-overview.adoc]: Addressed

**Pull request merged**